File: Source/burgers.py
# Chef is fond of burgers and decided to make as many burgers as possible.

# Chef has AA patties and BB buns. To make 11 burger, Chef needs 11 patty and 11 bun.
# Find the maximum number of burgers that Chef can make.


# Number of Test cases
t=int(input()) 


# Figure Out how to take input with just a space
# Patties and buns
a=int(input(""))
b=int(input(""))

i=1
while i<=t:
    if a<=b:
         print(a)
    elif a==b:
        print(a)
    elif b<=a:
        print(b)
    i=i+1

File: Source/burgers.cpp
#include <iostream>
using namespace std;
class burgers
{
    public:
    int i;
    int a,b; // Nuber of Patties and Buns
    int t; // Number of Test cases
    void check()
    {
        cin>>t;
        for (i=1;i<=t;i++)
        {
            cin>>a>>b;
            if(a<=b)
            {
                cout<<a;
            }
            else if(b<=a)
            {
                cout<<b;
            }
            else if (a==b)
            {
                cout<<a;
            }
        }
    }
};
int main()
{
    burgers obj;
    obj.check();
    return 0;
}


// Ans is correct